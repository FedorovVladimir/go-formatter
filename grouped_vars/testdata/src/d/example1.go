package main

var ( // want "grouped vars"
	a = "a"
)
