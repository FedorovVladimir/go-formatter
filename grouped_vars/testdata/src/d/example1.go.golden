package main

var a = "a"
